README.md

```diff
@@ -108,34 +130,34 @@ interface IBaseItem {
   median: number;
   max: number;
   /**
-   * values of the raw items used for rendering jittered background points 
+   * values of the raw items used for rendering jittered background points
    */
-  items?: number[];     
+  items?: number[];
 }
 
 interface IBoxPlotItem extends IBaseItem {
   q1: number;
   q3: number;
   /**
-   * list of box plot outlier values  
+   * list of box plot outlier values
    */
   outliers?: number[];
 }
 
 interface IKDESamplePoint {
```

Comment:
I had to look up what KDE means, maybe a short note in the readme that the Kernel density estimation is used here would help